web_src/less/features/projects.less

```diff
@@ -0,0 +1,77 @@
+.board {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  overflow-x: auto;
+  margin: 0 .5em;
+}
+
+.board-column {
+  background-color: var(--color-project-board-bg) !important;
+  border: 1px solid var(--color-secondary) !important;
+  margin: 0 .5rem !important;
+  padding: .5rem !important;
+  width: 320px;
+  height: 60vh;
+  overflow-y: scroll;
+  flex: 0 0 auto;
+  overflow: visible;
+  display: flex;
+  flex-direction: column;
+}
+
+.board-column-header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.board-label {
+  background: none !important;
+  line-height: 1.25 !important;
+}
+
+.board-column > .cards {
+  flex: 1;
+  display: flex;
+  margin: 0 !important;
+  padding: 0 !important;
+  flex-wrap: nowrap !important;
+  flex-direction: column;
```

Comment:
Regarding #15506, wouldn't that cause this issue again?
The solution I found was to remove this statement.
Why would it even be necessary if we don't even want them to be organized in columns?

Reply:
`flex-wrap: nowrap` should prevent any wrapping. `flex-direction` defaults to `row` meaning a horizontal flex direction which is not what we want. I need to double-check but I think the problematic `flex-wrap` is set in Fomantic UI which is why I put the override rule.

Reply:
So I played around this with this pretty long, it's a complex issue because Fomantic UI applies a lot of flexbox styles on those boxes and we have to fight those styles. Current solution seems to work acceptably.